science/nco/Portfile

```diff
@@ -79,4 +79,11 @@ variant openmp description {enable support for OpenMP} {
     compiler.fallback       macports-clang-7.0
 }
 
+variant esmf description {use ESMF (Earth System Modeling Framework)} {
+    configure.args-delete   --disable-esmf
+    configure.args-append   --enable-esmf
+    depends_lib-append      port:esmf
+    depends_run-append      port:esmf
```

Comment:
this makes no sense, it should be listed only once. If it is only needed at runtime, then use `depends_run`, if it also is needed at build time then it should be `depends_lib`